juniper_official/Routing-options/check-jsr-kkcm-conn.rule

```diff
@@ -0,0 +1,82 @@
+/*
+ * Report KKCM connection issues, if nonstop-routing feature is configured.
+ *
+ * If this rule reports an issue, make sure that nonstop-routing is indeed
+ * configured. Try removing this configuration and then reconfiguring the same.
+ */
+iceberg {
+    topic routing-options.nsr {
+        rule check-jsr-kkcm-conn {
+            synopsis "Check KKCM connections when NSR/JSR is enabled";
+            description "Report KKCM connection issues when NSR is configured";
+            sensor kkcm-sensor {
+                iAgent {
+                    file jsr_kkcm.yml;
+                    table checkJSRTable;
+                    frequency 5m;
+                }
+            }
+            field jsr {
+                sensor kkcm-sensor {
+                    path jsr;
+                }
+                description "JSR status - set when nonstop-routing is enabled";
+            }
+            field kkcm {
+                sensor kkcm-sensor {
+                    path kkcm;
+                }
+                description "This field is to parse number of KKCM connections";
+            }
+            trigger report-jsr-kkcm-health {
```

Comment:
1) Provide trigger description and synopsis detailing sensor info.
2) Trigger frequency is must as default frequency is 10s and trigger gets evaluated once in 10s. but here sensor frequecy is 10 mins hence trigger frequency can be greaterthan equal to 10 mins.
e.g. synopsis "Routing engine GRES kpi";
       description "sets health based on Routing engine CPU utilization for 15 minutes";

Reply:
Addressed.